models/tasks.js

```diff
@@ -51,7 +52,27 @@ const fetchTasks = async () => {
     const tasks = buildTasks(tasksSnapshot);
     const promises = tasks.map(async (task) => fromFirestoreData(task));
     const updatedTasks = await Promise.all(promises);
-    const taskList = updatedTasks.map((task) => {
+    const fetchTasksWithRdsAssigneeInfo = updatedTasks.map(async (task) => {
+      if (Object.keys(task).includes("github")) {
+        if (Object.keys(task.github.issue).includes("assignee")) {
+          return {
+            ...task,
+            github: {
+              ...task.github,
+              issue: {
+                ...task.github.issue,
+                assigneeRdsInfo: await getRdsUserInfoByGitHubUsername(task.github.issue.assignee),
+              },
```

Comment:
I think we should not be doing this in the model
This should happen outside the model in service or in the controller

Reply:
Done.